scripts/templates/nexus-repository-ha-operator-certified.clusterserviceversion.yaml

```diff
@@ -239,17 +285,23 @@ spec:
     | `statefulset.readinessProbe.periodSeconds`                  |  The interval for checking the Nexus Repository application for traffic readiness | 60|
     | `statefulset.readinessProbe.failureThreshold`               |  The maximum number of times to check the Nexus Repository application for traffic readiness | 6|
     | `statefulset.imagePullSecrets`                              |  The pull secret for private image registries | {}|
-    | `ingress.name`                                              |  The name of Nexus Repository Ingress object | nexus-ingress|
-    | `ingress.enabled`                                           |  Whether or not to create the Nexus Repository Ingress | "false"|
+    | `ingress.name`                                              |  The name of an Ingress object which listens for requests to Nexus Repository on port 80 | nexus-ingress|
+    | `ingress.enabled`                                           |  Whether or not to create the Nexus Repository Ingress | false|
     | `ingress.host`                                              |  Ingress host | "example.com"|
-    | `ingress.defaultRule`                                       |  Whether or not to add a default rule for the Nexus Repository Ingress which forwards traffic to a Service object  | "false"|
+    | `ingress.defaultRule`                                       |  Whether or not to add a default rule for the Nexus Repository Ingress which forwards traffic to a Service object  | false|
     | `ingress.additionalRules`                                   |  Additional rules to add to the ingress | null|
-    | `ingress.tls.enabled`                                       |  Whether or not TLS should be enabled for the ingress | "false"|
+    | `ingress.tls.enabled`                                       |  Whether or not TLS should be enabled for the ingress | false|
     | `ingress.tls.secretName`                                    | The name of a Secret object to store TLS secret in | tlsSecretName|
     | `ingress.dockerSubdomain`                                   |  Whether or not to add rules for docker subdomains | false|
     | `ingress.annotations`                                       |  Annotations for the ingress object | null |
     | `license.fileContentsBase64`                                |  Your Base64 encoded Nexus Repository license | nexus-repo-license (must be overridden via Kubernetes secret object which feeds into env variable variables defined in the operator.yaml . See above)|
```

Comment:
This line is confusing me. I get this is the license file - I'm confused by everything from "which feeds into env variable variables defined in the operator.yaml. See above" 

I'm not sure where above you mean that they should see, and the "variable variables" thing is confusing.

Reply:
https://github.com/sonatype/operator-nexus-repository/pull/5/commits/cb1e4341a5c89fe0ab9af8a585395f43e2dea54f